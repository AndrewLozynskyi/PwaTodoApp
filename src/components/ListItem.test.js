import React from 'react';
import { shallow } from 'enzyme';
import ListItem from './ListItem';

describe('ListItem', () => {
  it('renders without crashing', () => {
    const component = shallow(<ListItem />);
    expect(component).toMatchSnapshot();
  });

});
