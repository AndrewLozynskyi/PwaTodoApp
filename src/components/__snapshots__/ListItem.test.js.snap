// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem renders without crashing 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
